laterpay/compat.py

```diff
@@ -11,6 +11,6 @@ def encode_if_unicode(value, encoding='utf-8'):
     Encoding is done only if ``value`` is a ``unicode`` instance
     (utf-8 encoding is used as default).
     """
-    if six.PY2 and isinstance(value, unicode):
+    if six.PY2 and isinstance(value, six.text_type):
```

Comment:
OK, thanks, that makes sense now. If you could amend the commit message and list the functional changes in there, that would be great. As a reviewer here, I'm looking for something like for example "* don't pin basepython version anymore" and/or "don't run a separate `cov` job" etc.